scipy/stats/_continuous_distns.py

```diff
@@ -6121,19 +6122,82 @@ def _entropy(self, c):
     def fit(self, data, *args, **kwds):
         parameters = _check_fit_input_parameters(self, data, args, kwds)
         data, fshape, floc, fscale = parameters
-        if floc is None:
-            return super().fit(data, **kwds)
+
+        ndata = data.shape[0]
+
+        def get_shape(scale, location):
+            # The partial derivative w.r.t. the shape can be solved in
+            # closed form for the shape.
```

Comment:
A "partial derivative" is not an equation, so it cannot be "solved". 
You mean that the equation `<partial derivative w.r.t. shape> = 0` can be solved. The name for this sort of thing is "first-order necessary condition".
```suggestion
            # The first-order necessary condition on `shape` can be solved in
            # closed form
```